Sources/WalletConnectPush/Types/PushMessage.swift

```diff
@@ -0,0 +1,16 @@
+
+import Foundation
+
+public struct PushMessage: Codable {
```

Comment:
All params is required?

Reply:
so far, yes but it may be a good suggestion to change specs